packages/vscode-extension/package.json

```diff
@@ -162,7 +167,19 @@
         {
           "id": "teamsfx-help-and-feedback",
           "name": "Help and feedback",
-          "when": "fx-extension.isTeamsFx && !fx-extension.canUpgradeV3",
+          "when": "fx-extension.isTeamsFx && !fx-extension.canUpgradeV3 && !fx-extension.isV3Enabled",
+          "visibility": "visible"
+        },
+        {
+          "id": "teamsfx-help",
+          "name": "Help",
+          "when": "fx-extension.isTeamsFx && fx-extension.isV3Enabled",
```

Comment:
I think `!fx-extension.canUpgradeV3` should be added here so that the view will not appear in welcome view.